### ISSUE TITLE
Error when building in a directory with spaces in it

### ISSUE THREAD

**Issue by pfrazee** [mafintosh/utp-native#7]
Hey, I'm getting this in OSX when I run a rebuild, in a directory with spaces in the path. In this case, the dir is `/Users/paulfrazee/Library/Application Support/Beaker Browser/`

```
> utp-native@1.3.0 install /Users/paulfrazee/Library/Application Support/Beaker Browser/node_modules/utp-native
> prebuild --install

  CXX(target) Release/obj.target/libutp/deps/libutp/utp_internal.o
../deps/libutp/utp_internal.cpp:1969:34: warning: comparison of integers of
      different signs: 'int' and 'uint32' (aka 'unsigned int') [-Wsign-compare]
                if (conn->mtu_probe_seq && seq == conn->mtu_probe_seq) {
                                           ~~~ ^  ~~~~~~~~~~~~~~~~~~~
1 warning generated.
  CXX(target) Release/obj.target/libutp/deps/libutp/utp_utils.o
  CXX(target) Release/obj.target/libutp/deps/libutp/utp_hash.o
  CXX(target) Release/obj.target/libutp/deps/libutp/utp_callbacks.o
  CXX(target) Release/obj.target/libutp/deps/libutp/utp_api.o
  CXX(target) Release/obj.target/libutp/deps/libutp/utp_packedsockaddr.o
  LIBTOOL-STATIC Release/utp.a
  CXX(target) Release/obj.target/utp/src/utp_uv.o
clang: error: no such file or directory: 'Support/Beaker'
clang: error: no such file or directory: 'Browser/node_modules/utp-native/deps/libutp'
make: *** [Release/obj.target/utp/src/utp_uv.o] Error 1
```

Somewhere, a clang command is invoked without quotes around the path. I scanned through the code but don't know enough about node-gyp to spot where (sorry, no PR).

**Comment by mafintosh** [comment]
Seems related to this unfortunately, https://github.com/nodejs/node-gyp/issues/65

**Comment by pfrazee** [comment]
Well, that's absurd!

These guys seem to think they solved it: https://github.com/lovell/sharp/issues/533. Looks like they used relative paths. I'll try the same on this

**Comment by mafintosh** [comment]
fixed in 1.3.1 using that same trick
